### DISCUSSION TITLE
GraphQL: not able to query for eventCount similar to built-in entryCount/assetCount

### DISCUSSION THREAD

**Discussion by denisyilmaz** [solspace/craft-calendar#165] [Feature Requests]
We have a list of current and upcoming events and want to show a "explore all XXX events in the archive" button the the list.
Craft CMS has a built in `entryCount` field that returns the number of items fitting the parameters.

Would it be possible to add a similar field `eventCount` where we would be able to query for the number of past events?

Example
```gql
eventCount(siteId: [$siteId], rangeEnd: "now", startsAfter:"now - 12 month")
```
... would return the number of events that ended before today and started sometime in the past 12 month.

**Comment by seandelaney** [comment]
Hopefully resolved in next release

https://github.com/solspace/craft-calendar/pull/422
